src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -288,20 +302,256 @@ private static ProjectItemGroupElement CreateItemGroup(Project project, string f
             }
         }
 
+        public Dictionary<string, Tuple<IEnumerable<PRPackage>, IEnumerable<PRPackage>>> GetPackageReferencesForList(
+            Project project, string projectPath, IList<string> userInputFrameworks, bool transitive)
+        {
+            //Path to the assets file
+            var assetsPath = project.GetPropertyValue(ASSETS_FILE_PATH_TAG);
+
+            //If the property or file were not found, print an error message and return
+            if (assetsPath == "" || !File.Exists(assetsPath))
```

Comment:
This checks if the assets file was not found for any reason whether it's because the project is `packages.config` or restore was not run. I do not know if there is a way to say for sure which one it is.

Reply:
`RestoreProjectStyle` does not always have a value actually. I'll look at `TargetFramework` and see if I can figure out how to do this

Reply:
There's a target that evaluates the RestoreProjectStyle. 

The same should be used for consistency sake.